### ISSUE TITLE
Remember shuffle mode

### ISSUE THREAD

**Issue by joelmo** [TeamNewPipe/NewPipe#1795]
- [x] I carefully read the [contribution guidelines](https://github.com/TeamNewPipe/NewPipe/blob/HEAD/.github/CONTRIBUTING.md) and agree to them.
- [x] I checked if the issue/feature exists in the latest version.
- [ ] I did use the [incredible bugreport to markdown converter](https://teamnewpipe.github.io/CrashReportToMarkdown/) to paste bug reports.

I use NewPipe for listening to music and I use shuffle mode alot. Therefore I think it would be nice to remember shuffle mode, maybe this can be set as an option in settings. What do you think?

I usually start a playlist from bookmarks, and from there I have to navigate to the player queue and turn on shuffle mode.

**Comment by theScrabi** [comment]
I don't think this needs to be enabled in settigs. We could just make the shuffel toggel persistend.

**Comment by agarwalakash06** [comment]
I am working on this.
Also I think if we are persisting the shuffle mode, then we should also persist the repeat mode.
What are your views?

**Comment by PeterHindes** [comment]
That is a viable solution. We might also look at implementing double tap on the background button to shuffle.

**Comment by Stypox** [comment]
Adding another gesture to the background button (which aready has two: click and long click) does not seem right, also, Android discourages double-click gestures since they are typical on computers but feel strange on phones.

**Comment by Guan-1023** [comment]
Hi! 
I'm a student from the Australian National University (ANU), and as part of a group assignment for one of our courses, we’re asked to contribute to an open-source project by resolving a real issue.
I’d love to work on this issue and submit a pull request with the fix.
Could you please let me know if it's okay for me to take this one?
Thank you!
